packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxis.ts

```diff
@@ -14,9 +13,13 @@ import { getAxisValue } from './getAxisValue';
 import { getSVGPoint } from '../../../getSVGPoint';
 import { selectorChartsInteractionAxis } from '../useChartInteraction';
 
-export const useChartCartesianAxis: ChartPlugin<
-  UseChartCartesianAxisSignature<ChartSeriesType>
-> = ({ params, store, seriesConfig, svgRef, instance }) => {
+export const useChartCartesianAxis: ChartPlugin<UseChartCartesianAxisSignature<any>> = ({
```

Comment:
`<any>` here seems better than the previous `ChartSeriesType`, since `ChartSeriesType` means `all of these`, which causes conflicts.

I believe the main issue in in the `ChartPlugin` signature. As a function with extra props (getInitialState,models,etc) is a bit complex to type.

Reply:
Note for myself tomorrow: Test if `CartesianChartsSeriesType` make it works.

Otherwise I agree that the any is better than the override

Reply:
After verification it does not work

Reply:
Yeah it wont work with anything other than "any". To make it work we would have to make it work with `useChartCartesianAxis<'bar'>`, but currently only the type is generic.

This means that the we can make the type specific, but not the implementation. So `a: UseChartCartesianAxis<'bar'> = useChartCartesianAxis` gives an error, because `useChartCartesianAxis = useChartCartesianAxis<'bar'> | useChartCartesianAxis<'pie'> | ...`